deploy/charts/harvester/values.yaml

```diff
@@ -32,7 +32,7 @@ kubevirt-operator:
     operator:
       image:
         repository: registry.suse.com/harvester-beta/virt-operator
-        tag: &kubevirtVersion 0.40.0-1
+        tag: &kubevirtVersion 0.40.0-2
```

Comment:
will also need to update the image tag in the https://github.com/harvester/harvester/blob/master/deploy/charts/harvester/questions.yaml#L146, thanks

Reply:
missed that one... updated.